Blockzilla/Tracking Protection/TrackingProtectionViewController.swift

```diff
@@ -233,7 +233,7 @@ class TrackingProtectionViewController: UIViewController {
         
         if case let .browsing(browsingStatus) = state,
            let baseDomain = browsingStatus.url.baseDomain,
-           let url = browsingStatus.faviconURL {
+           let _ = browsingStatus.faviconURL {
```

Comment:
`faviconURL` is not used anymore. The check can be removed and also the variable.